.github/ci-nightly-test.sh

```diff
@@ -14,12 +14,12 @@ module load python3
 
 version=$(cat $TMPDIR/eccodes/VERSION)
```

Comment:
Please put a comment that this script expects the env var GH_TOKEN to be there!!!